bin/cml.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+const { basename } = require('path');
+const { pseudoexec } = require('pseudoexec');
+
+const which = require('which');
+const winston = require('winston');
+const yargs = require('yargs');
+
+const configureLogger = (level) => {
+  winston.configure({
+    format: process.stdout.isTTY
+      ? winston.format.combine(
+          winston.format.colorize({ all: true }),
+          winston.format.simple()
+        )
+      : winston.format.json(),
+    transports: [
+      new winston.transports.Console({
+        handleExceptions: true,
+        handleRejections: true,
+        level
+      })
+    ]
+  });
+};
+
+const runPlugin = async ({ $0: executable, command }) => {
+  try {
+    if (command === undefined) throw new Error('no command');
+    const path = which.sync(`${basename(executable)}-${command}`);
+    const parameters = process.argv.slice(process.argv.indexOf(command) + 1); // HACK
+    process.exit(await pseudoexec(path, parameters));
+  } catch (error) {
+    winston.debug(error);
+    yargs.showHelp();
```

Comment:
```suggestion
    yargs.showHelp();
    winston.debug(error);
```

Show the error after the help string, like with the other kinds of error.